code/game/area/areas.dm

```diff
@@ -124,6 +124,9 @@ var/area/space_area
 						a.cancelAlarm("Power", src, source)
 					else
 						a.triggerAlarm("Power", src, cameras, source)
+			for (var/obj/item/device/pager/P in pager_list)
+				if(state != 1)
```

Comment:
did you mean muted?

Reply:
This check isn't even needed here - removed